src/components/TravelSection.tsx

```diff
@@ -59,31 +59,79 @@ const TravelSection = () => {
     window.open(link, '_blank', 'noopener,noreferrer');
   };
 
+  const handleKeyDown = (event: React.KeyboardEvent) => {
+    if (event.key === 'ArrowLeft') {
+      event.preventDefault();
+      prevTravel();
+    } else if (event.key === 'ArrowRight') {
+      event.preventDefault();
+      nextTravel();
+    }
+  };
+
+  const formatPrice = (price: number) => {
+    return price.toLocaleString('ko-KR');
+  };
+
   return (
-    <div className={styles.travelSection}>
-      <button className={`${styles.navButton} ${styles.navButtonPrev}`} onClick={prevTravel}>
-        <img src={chevronLeft} className={styles.navButtonIcon} />
+    <div
+      className={styles.travelSection}
+      role="region"
+      aria-label="지금 떠나기 좋은 여행 상품 캐로셀"
+      onKeyDown={handleKeyDown}
+      tabIndex={0}
+    >
```

Comment:
```suggestion
    <section
      className={styles.travelSection}
      aria-label="지금 떠나기 좋은 여행 상품 캐로셀"
      onKeyDown={handleKeyDown}
      tabIndex={0}
    >
```
이전 코멘트에서 잠깐 언급했는데 저도 이번에 학습하면서 알게 되었는데 이 부분에 `section` 태그를 사용하면 `section` 태그 내부에 `role="region"`이 있어서 속성을 직접 주지 않아도 동일한 기능을 할 수 있다고 하네요~

추가적으로 상호작용이 불가능한 태그에 `tabIndex={0}` 주신 이유가 궁금합니다!

Reply:
section 자체적으로 role="region" 속성이 있다는 것도 알아갑니다! 😄

제 생각으로는 section 영역을 나누는 느낌이다보니 더 상위에 있어야 맞다고 생각했아요. 그래서 TravelSection의 젤 상위 태그를 div로 주고 tabIndex로 탭으로 포커스가 가능하도록 했습니다.

Reply:
section 태그는 영역을 나누기 때문에 div 태그보다 더 위에 있는 것이 좋아보인다는 의견이 합리적인 것 같네요~👍👍👍